src/reporter.js

```diff
@@ -38,11 +38,29 @@ var resultsFolder;
 
 var uniqueFileId;
 
+function getSettings(options) {
+
+  var config = {
```

Comment:
You should either name the function `getConfig` or change the variable to `settings`. However, since you already have `loadConfiguration`, the latter is better...
And similarly, further down when you call that function.